packages/mask/src/plugins/Pets/SNSAdaptor/Drag.tsx

```diff
@@ -9,7 +9,7 @@ interface StateProps {
     rel: any
```

Comment:
Please don't rename a file that only differs in the casting. It will cause git not to rename this file on the case-insensitive file system.

Reply:
I won't rename any file in the future